bootstrap/sql/mysql/create_tables.sql

```diff
@@ -352,3 +352,31 @@ CREATE TABLE IF NOT EXISTS component (
   timestamp BIGINT,
   PRIMARY KEY (id)
 );
+
+CREATE TABLE IF NOT EXISTS topology_test_run_case (
+  id BIGINT AUTO_INCREMENT NOT NULL,
+  topologyId BIGINT NOT NULL,
+  versionId BIGINT NOT NULL,
+  records TEXT NOT NULL,
+  timestamp BIGINT,
+  PRIMARY KEY (id),
+  FOREIGN KEY (topologyId) REFERENCES topology(id),
+  FOREIGN KEY (versionId) REFERENCES topology_version(id)
+);
+
+CREATE TABLE IF NOT EXISTS topology_test_run_histories (
+  id BIGINT AUTO_INCREMENT NOT NULL,
+  topologyId BIGINT NOT NULL,
```

Comment:
If we have a fk reference to topology_test_run_case id then we can get rid of topologyId here since an entry in that table will provide topologyId and version. If its kept here only for convenience i am fine with it